src/view3d.cpp

```diff
@@ -226,8 +226,14 @@ std::unique_ptr<QGLFramebufferObject> View3D::allocIncrementalFramebuffer(int w,
     fboFmt.setAttachment(QGLFramebufferObject::Depth);
     fboFmt.setSamples(fmt.samples());
     //fboFmt.setTextureTarget();
-    return std::unique_ptr<QGLFramebufferObject>(
-        new QGLFramebufferObject(w, h, fboFmt));
+    QGLFramebufferObject* fbo = new QGLFramebufferObject(w, h, fboFmt);
+    switch (fbo->attachment())
+    {
+        case QGLFramebufferObject::NoAttachment:
+            g_logger.error("%s", "Failed to attach FBO depth buffer.");
+            break;
```

Comment:
Yeah, I considered logging the other possible attachment types, but decided against it in the end.
I don't mind either way.